control-plane-agent/api/proxy/proto/mediaproxy/mediaproxy.pb.go

```diff
@@ -0,0 +1,1965 @@
+// SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Why you include generated file ? 
Is it auto-generated each time build is executed?

Reply:
Yeah, it is executed. We can exclude this generated file.

Reply:
Fixed